cmake/libs/libraft.cmake

```diff
@@ -14,9 +14,10 @@
 # the License.
 
 add_definitions(-DKNOWHERE_WITH_RAFT)
+add_definitions(-DRAFT_EXPLICIT_INSTANTIATE_ONLY)
```

Comment:
See above. This enforces that we only use instantiations that are specifically optimized for in RAFT.